ci/cleanup_namespaces

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+own_dir="$(readlink -f "$(dirname "${0}")")"
+repo_root="${own_dir}/.."
+
+source "${own_dir}/lib.sh"
+
+install_kubectl
```

Comment:
Now we have a second file containing these ugly install xyz commands. I would prefer to ensure that these tools are available in our environment (concourse image) and get rid of those commands.